docs/deployment/ssl.md

```diff
@@ -0,0 +1,165 @@
+---
+title: "SSL"
+draft: false
+slug: SSL
+url: /deployment/ssl
+---
+# SSL
+
+Convox will automatically generate a valid SSL certificate for your service using Let's Encrypt. By default, Convox handles SSL certificate generation via HTTP-01 and email validation methods.
+
+If you specify a custom `domain:` attribute for your service, Convox will automatically generate and validate an SSL certificate using Let's Encrypt's default HTTP-01 challenge. You may also receive an email for domain ownership verification during this process.
+
+## Pre-generate your certificate
+
+To minimize delays during your first deployment, you can pre-generate your SSL certificate. This ensures your service is available without waiting for the certificate issuance process.
+
+```
+$ convox certs generate "*.example.org" "myapp.example.org"
```

Comment:
this will generate self cert, not letsencrypt one
for letsencrypt one you have to specify issuer
```
--issuer letsencrypt
```

Reply:
fixed